index.html

```diff
@@ -53,20 +58,62 @@ <h2 class="symbols-title">Compliance Categories</h2>
                 </div>
               </div>
             </div>
-
         </div>
 
-        <h2 class="secondary-title">Department Progress</h2>
-        <div class="container">
-        </div>
+        <div class="report-header">
+            <h2 class="secondary-title">Department Progress</h2>
 
+            <script id="metadata-template" type="text/x-handlebars-template">
+                <p><a href="https://data.austintexas.gov/resource/8qty-vat2" target="_blank">Source data</a> last updated {{updated}}</p></i>
+            </script>
+        </div>
 
+        <div class="report-wrapper"><div class="report-container"></div></div>
 
+        <script id="card-template" type="text/x-handlebars-template">
```

Comment:
fancy! :nail_care: 
Handlebars template is a way nicer option than my hacky HTML block variable template inside main.js. I started the original one with @sharlalikesyou so I wanted to keep it simple and minimize outside libraries, but this makes the best sense.